src/cursor/cur_file.c

```diff
@@ -698,6 +698,8 @@ __curfile_cache(WT_CURSOR *cursor)
 
     WT_TRET(__wt_cursor_cache(cursor, cbt->dhandle));
     WT_TRET(__wt_session_release_dhandle(session));
+
+    API_END_STAT(session, ret, cursor_cache);
```

Comment:
Adding this change will double count the `cursor_cache` failure. The `cursor_cache` failure is already counted inside the `__wt_cursor_cache` function. Remove the newly added code in this function.

Reply:
Yes, I agree with you.